packages/rancher-logging/rancher-logging.patch

```diff
@@ -107,4 +107,10 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +  cattle:
 +    systemDefaultRegistry: ""
 +
-+fluentbit_tolerations: []
++fluentbit_tolerations:
++  - key: node-role.kubernetes.io/controlplane
++    value: "true"
++    effect: NoSchedule
++  - key: node-role.kubernetes.io/etcd
++    value: "true"
++    effect: NoExecute
```

Comment:
This is specifically the settings for rke ?

Reply:
yes, this counters the taints set on rke nodes